src/dialects/abstract/query-interface.js

```diff
@@ -289,7 +290,7 @@ export class QueryInterface {
     for (const tableName of tableNames) {
       // if tableName is not in the Array of tables names then don't drop it
       if (!skip.includes(tableName.tableName || tableName)) {
-        await this.dropTable(tableName, { ...options, cascade: true });
+        await this.dropTable(tableName, options);
```

Comment:
Something like this? https://github.com/sequelize/sequelize/pull/15235/commits/2f17e6e381afd0246bc6e0bf01fadf48649918f6

Reply:
No because with this version if the user calls `queryInterface.dropTable(table, { cascade: true })` on a dialect that doesn't support cascade, they won't be warned

It's this that needs changing:

```ts
// line 293, _dropAllTables
await this.dropTable(tableName, { cascade: this.queryGenerator.dialect.supports.dropTable.cascade ? true : undefined, ...options });
```

Reply:
Wasn't sure how to do the unit test with most dialects expecting an error and one dialect to return a query but I think I've done it okay; https://github.com/sequelize/sequelize/pull/15235/commits/11939bceb72a3bf5db2804415e0bec5b649a350a